### PR TITLE
feat(rn) remove use of externalAPIScope

### PR DESCRIPTION
**Pull request by saghul**
Use the system broadcasting mechanism instead.

On Android I took the chance and removed the no longer needed
BaseReactView and implemented it on JitsiMeetView instead.

### CODE REVIEW

File: android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetView.java [hunk 1]
```diff
@@ -109,10 +106,19 @@ public JitsiMeetView(Context context, AttributeSet attrs, int defStyle) {
...
 
-    @Override
+    /**
+     * Releases the React resources (specifically the {@link ReactRootView})
+     * associated with this view.
+     *
+     * MUST be called when the {@link Activity} holding this view is destroyed,
+     * typically in the {@code onDestroy} method.
+     */
     public void dispose() {
```
Comment (tmoldovan8x8) [inline android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetView.java:116]: should we make this private?

File: android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetView.java [hunk 2]
```diff
@@ -16,36 +16,33 @@
...
 
     /**
-     * The {@code Method}s of {@code JitsiMeetViewListener} by event name i.e.
-     * redux action types.
+     * Background color used by {@code BaseReactView} and the React Native root
+     * view.
      */
-    private static final Map<String, Method> LISTENER_METHODS
-        = ListenerUtils.mapListenerMethods(JitsiMeetViewListener.class);
+    private static int BACKGROUND_COLOR = 0xFF111111;
```
Comment (tmoldovan8x8) [inline android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetView.java:40]: This can also be final

**Comment by tmoldovan8x8** [comment]
LGTM. Just some tiny comments

File: android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetView.java (hunk 1 above)
Comment (saghul) [reply android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetView.java:116]: No, this is necessary for those integrating the view in their own activity, we do support that.

File: android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetView.java (hunk 2 above)
Comment (saghul) [reply android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetView.java]: 👍

**Review by tmoldovan8x8 — APPROVED** [review]

**Pull request merged**